build/templates/errors.rst.mako

```diff
@@ -64,6 +64,17 @@ ${helper.get_rst_header_snippet('SelfTestError', '-')}
         An error due to a failed self-test
 
 
+% endif
+% if 'DriverTooNewError' in extra_errors_used:
+${helper.get_rst_header_snippet('DriverTooNewError', '-')}
+
+    .. py:currentmodule:: ${module_name}.errors
+
+    .. exception:: DriverTooNewError
+
+        An error due to the driver runtime being too new compared to the Python module
```

Comment:
```suggestions
An error due to the driver runtime being too new for the Python module
```

Reply:
I have updated the docs here as well as the class comment in errors.py.mako.